packages/my-react-package/src/MetaWallet.tsx

```diff
@@ -56,15 +59,39 @@ class MetaWallet {
     attachGasStation = (gas_station: string, api_key: string): void => {
         this.gasStations[gas_station] = api_key;
     }
+    /**
+         * Attach a gas station with this wallet to so it can be used 
+         * as a relayer later
+         * 
+         * @param {string} txBuilder - name of the TX Builder
+         * @param {string} api_key - The API key (endpoint) of the TX Builder
+         * @returns {void}
+         */
+    attachTXBuilder = (TXBuilder: string, api_key: string): void => {
+        this.TXBuilders[TXBuilder] = api_key;
+    }
 
     // @TODO: attach an API endpoint to call when building the execution transaction (using biconomy)
 
-    // @TODO: get the right way to sign the transaction based on biconomy
-    async getSignedTX(argsTypes: Array<string>, argsValues: Array<any>, functionName: string, functionParamsInterface: string) {
+    /**
+             * sign the transaction
+             * 
+             * @param {string} scwAddress - Address of the smart contract wallet 
+             * @param {string} chainId - chain Id of the transaction
+             * @param {any} safeTxBody - the transaction built by TXBuilder 
+             * @returns {Promise<string>} the signed transaction
+             */
+    async getSignedTX(scwAddress: string, chainId: string, safeTxBody: BuildExecTransaction):Promise<string>{
```

Comment:
Let's put the `scwAddress` as an attribute of the MetaWallet class instead of passing it as a parameter.